src/__tests__/within.test.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React from 'react';
+import { View, Text, TextInput } from 'react-native';
+import { render } from '..';
+import { within } from '../within';
```

Comment:
we import from index.js for a reason – `within` is not exposed to the user currently, please import it there and then, in this test, from there :)

Reply:
I've added proper index.js level export and import here.